contracts/contracts/strategies/BaseCurveAMOStrategy.sol

```diff
@@ -133,14 +141,17 @@ contract BaseCurveAMOStrategy is InitializableAbstractStrategy {
      * well within that abstraction.
      * @param _rewardTokenAddresses Address of CRV
      * @param _assets Addresses of supported assets. eg WETH
+     * @param _maxSlippage Maximum slippage allowed for adding/removing liquidity from the Curve pool.
      */
     function initialize(
         address[] calldata _rewardTokenAddresses, // CRV
-        address[] calldata _assets // WETH
+        address[] calldata _assets, // WETH
+        uint256 _maxSlippage
     ) external onlyGovernor initializer {
         require(_assets.length == 1, "Must have exactly one asset");
         require(_assets[0] == address(weth), "Asset not WETH");
```

Comment:
nit: why not just make it a hardcoded constant/variable instead of passing it as an argument when we know what exactly it should have?

Reply:
I think simplifying deployment is safe. 
Changed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2384/commits/4e2e80c7532397747ac9a705fac0dd4279a11985.

Reply:
Fixed in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2384/commits/4e2e80c7532397747ac9a705fac0dd4279a11985